packages/runtime/src/next/api-handler/handlers/proxy-draft-mode.test.ts

```diff
@@ -0,0 +1,127 @@
+import { NextRequest } from 'next/server'
+import proxyDraftMode from './proxy-draft-mode'
+import { env } from 'process';
+
+jest.mock('next/headers', () => ({
+  draftMode: jest.fn().mockReturnValue({ enable: jest.fn(), disable: jest.fn() }),
+  cookies: jest.fn().mockReturnValue({ get: jest.fn() }),
+}))
+
+describe('ForceHTTP proxyDraftMode URL handling', () => {
+  const mockApiKey = 'test-api-key'
+  let originalFetch: typeof global.fetch
+
+  beforeAll(() => {
+    originalFetch = global.fetch
+    env.FORCE_HTTP = 'true'
+    global.fetch = jest.fn().mockResolvedValue(new Response())
```

Comment:
I think we should avoid using Jest mocks as much as possible as they cause us to test too many implementation details IMO. I think we won't have a choice mocking certain things like `draftMode()` and `cookies()`, but even those, I would prefer if there were testing utilities provided by Next.js we could use or if our tests actually ran the Next.js dev server and sent requests to it to verify the request and response payloads.